_includes/head.html

```diff
@@ -13,6 +13,31 @@
     <meta name="description" content="{{page.description | escape }}" />
     <meta property="og:description" content="{{page.description | escape }}" />
     <meta name="twitter:description" content="{{page.description | escape }}" />
+    {% if page.category == "ERC" %}
```

Comment:
I kinda feel like we're at the point where we should do something like:

```liquid
{% if page.category == "ERC" %}
    {% assign prefix = "ERC" %}
{% else %}
    {% assign prefix = "EIP" %}
{% endif %}
```

Reply:
Yea, but this isn't the place to do that.